docs/metrics/accelerate.md

```diff
@@ -82,12 +91,12 @@ number of deployments that caused incidents / total number of deployments
 
 Pulse uses the average value over a period of time to display aggregate change failure rates.
 
-!!! note
-    We decided to avoid requiring a relationship between **incidents** and **deployments** or **changes** to simplify the data model reported.
+{%
+   include-markdown "accelerate.md"
+   start="<!--match-incident-deployment-start-->"
+   end="<!--match-incident-deployment-end-->"
+%}
```

Comment:
I decided to use the plugin include-markdown so we can include the note on the two sections without actually having to repeat the same text.